gradle.properties

```diff
@@ -1,6 +1,17 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=1.0.6-SNAPSHOT
+slf4jVersion=1.7.30
+testngVersion=6.14.3
+javaMailVersion=1.6.2
+greenmailVersion=1.5.11
+mimepullVersion=1.9.11
+puppycrawlCheckstyleVersion=8.18
+checkstyleToolVersion=7.8.2
+githubSpotbugsVersion=4.0.5
+githubJohnrengelmanShadowVersion=5.2.0
+underCouchDownloadVersion=4.0.4
+researchgateReleaseVersion=2.8.0
 ballerinaLangVersion=2.0.0-alpha2-SNAPSHOT
```

Comment:
We have to bump the ballerina lang version and all the stdlib versions.

Reply:
Bumped.